types/node/globals.d.ts

```diff
@@ -664,7 +664,7 @@ declare namespace NodeJS {
     type ExitListener = (code: number) => void;
     type RejectionHandledListener = (promise: Promise<any>) => void;
     type UncaughtExceptionListener = (error: Error) => void;
-    type UnhandledRejectionListener = (reason: {} | null | undefined, promise: Promise<any>) => void;
+    type UnhandledRejectionListener = (reason: unknown, promise: Promise<unknown>) => void;
```

Comment:
We cannot use `unknown` as. we still have to support TS 2.8.

Reply:
@SimonSchick can we use any instead ?

Reply:
Well yes, I always deeply disliked the ` {} | null | undefined` workaround and the `poor mans unknown` approach, it was more confusing than anything.